roles/infrastructure/defaults/main.yml

```diff
@@ -28,6 +28,7 @@ infra__vpc_public_subnets_suffix:   "{{ common__vpc_public_subnets_suffix }}"
 
 # Infra
 infra__type:                        "{{ common__infra_type }}"
+infra__tunnel:                      "{{ common__tunnel }}"
```

Comment:
What do we think about defaulting the tunnel to True? That seems the configuration more in line with our philosophy of 'working OOTB and as secure as possible' @wmudge

Reply:
Only way we can allow this is if `public_endpoint_access` is available on GCP/Azure, right?

Reply:
We could do this. In GCP/Azure, that option will be ignored. 
This will be the case for those providers even if you set it to true in definition file.

Reply:
Wanted to make the switch once all the networking changes are in and we've tried it enough times.

Reply:
Agreed. We can put the feature into this release, and make it the default once it has bedded in.